visual-dotnet/SauceLabs.Visual/VisualClient.cs

```diff
@@ -193,17 +205,21 @@ private async Task<VisualBuild> FindBuildById(string buildId)
         /// <returns>a <c>VisualBuild</c> instance</returns>
         private async Task<VisualBuild> CreateBuild(CreateBuildOptions? options = null)
         {
+            var projectName = options?.Project ?? Assembly.GetExecutingAssembly().FullName;
```

Comment:
Base on what we saw I think its safer to use "Unknown" rather than executing assembly - wdyt?

Reply:
We should remove it, and probably keep default behaviour.